requirements/edx/github.txt

```diff
@@ -91,6 +91,7 @@ git+https://github.com/edx/xblock-utils.git@v1.0.2#egg=xblock-utils==1.0.2
 git+https://github.com/edx/edx-user-state-client.git@1.0.1#egg=edx-user-state-client==1.0.1
 git+https://github.com/edx/xblock-lti-consumer.git@v1.0.9#egg=xblock-lti-consumer==1.0.9
 git+https://github.com/edx/edx-proctoring.git@0.12.21#egg=edx-proctoring==0.12.21
+git+https://github.com/edx/edx-django-release-util.git@0.0.3#egg=edx-django-release-util==0.0.3
```

Comment:
Install this from PyPI: https://github.com/edx/course-discovery/blob/bdbdb5b957b8b9d6360d816916dd22d520856ba1/requirements/base.txt#L20